libpod/util.go

```diff
@@ -148,3 +152,36 @@ func JSONDeepCopy(from, to interface{}) error {
 	}
 	return json.Unmarshal(tmp, to)
 }
+
+func dpkgVersion(path string) string {
+	output := unknownPackage
+	cmd := exec.Command("/usr/bin/dpkg", "-S", path)
+	if outp, err := cmd.Output(); err == nil {
+		output = string(outp)
+	}
+	return strings.Trim(output, "\n")
+}
+
+func rpmVersion(path string) string {
+	output := unknownPackage
+	cmd := exec.Command("/usr/bin/rpm", "-q", "-f", path)
+	if outp, err := cmd.Output(); err == nil {
+		output = string(outp)
+	}
+	return strings.Trim(output, "\n")
+}
+
+func packageVersion(program string) string {
+	if out := rpmVersion(program); out != unknownPackage {
+		return out
+	}
+	return dpkgVersion(program)
+}
+
+func programVersion(mountProgram string) (string, error) {
+	output, err := utils.ExecCmd(mountProgram, "--version")
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSuffix(strings.Replace(output, "\n", ", ", 1), "\n"), nil
```

Comment:
I like trimming the trailing newline but `strings.Replace(output, "\n", ", ", 1)` may not be applicable other programs (i.e., when there's only one line). Can we change it to:
```golang
return strings.TrimSuffix(output, "\n"), nil
```
?

Reply:
That's still present. @rhatdan, any comments?